File: PRAC/W11/devowel_file.py
#!/usr/bin/python

import sys
import re

file = open(sys.argv[1])
array = []
for line in file:
	line = re.sub(r"[aeiouAEIOU]", "", line)
	array.append(line)
file.close()

# remove file contents
file = open(sys.argv[1], 'w')
file.close()

# append devoweled lines
file = open(sys.argv[1], 'a')
for line in array:
	file.write(line)

File: PRAC/W11/devowel_file.pl
#!/usr/bin/perl

$file = $ARGV[0];

open F, '<', $file;

@array;
while ($line = <F>) {
	$line =~ s/[aeiouAEIOU]//g;
	push(@array, $line);
}
close F;

open F, '>', $file;
close F;

open F, '>>', $file;

foreach $item (@array) {
	print F $item;
}

